identify

```diff
@@ -16,8 +16,13 @@ fi
 
 arg="$1"
 
-if [[ -f "$arg" ]]; then
-  libc=$arg
+if [ "${1:0:1}" != "/" ]; then    # this is a relative path
```

Comment:
instead of making that decision post-chdir based on a heuristic, please use `realpath` before chdir:

```
if [ -f "$arg" ]; then
  arg="$(realpath "$arg")"
fi
```

Reply:
sorry, should be `readlink -f`, I always confuse the two

Reply:
done